### PR TITLE
Feature/eui 246

### PR DESCRIPTION
**Pull request by udaydenduluri33**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: api/lib/config/index.ts [hunk 1]
```diff
@@ -30,15 +30,12 @@ const configs = {
 export const configEnv = process ? process.env.PUI_ENV || 'local' : 'local'
 export const config = { ...application, ...configs[configEnv].default }
 
-
+export default { ...config }
+console.log(config)
```
Comment (adnanakgun) [inline api/lib/config/index.ts]: Console message

File: api/organisation/index.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import * as express from 'express'
+import { generateToken } from '../auth/serviceToken'
+import { config } from '../lib/config'
+import { http } from '../lib/http'
+
+
+async function handleOrganisationsRoute(req: express.Request, res: express.Response, next: express.NextFunction) {
+    try {
+        const response = await http.get(`${config.services.rdProfessionalApi}/organisations?status=ACTIVE`)
+        console.log(response.data)
```
Comment (adnanakgun) [inline api/organisation/index.ts]: Console log

File: api/services/serviceAuth.ts [hunk 3]
```diff
@@ -16,8 +17,8 @@ const logger = log4jui.getLogger('service auth')
 export async function postS2SLease() {
     const configEnv = process ? process.env.PUI_ENV || 'local' : 'local'
     let request: AxiosResponse<any>
-    console.log('test2:', configEnv)
-    if (configEnv !== 'local') {
+    console.log('PUI_ENV is now:', configEnv)
```
Comment (adnanakgun) [inline api/services/serviceAuth.ts]: Console log

Comment (udaydenduluri33) [reply api/services/serviceAuth.ts]: Done

File: api/organisation/index.ts (hunk 2 above)
Comment (udaydenduluri33) [reply api/organisation/index.ts]: Done

File: api/lib/config/index.ts (hunk 1 above)
Comment (udaydenduluri33) [reply api/lib/config/index.ts]: Done

File: src/org-manager/models/organisation.ts [hunk 4]
```diff
@@ -1,15 +1,43 @@
...
+  dxAddress: [OrganisationDxAddress]
+  }
+
+export interface OrganisationDxAddress {
+      dxNumber: string,
+      dxExchange: string
+  }
+
+export interface OrganisationSuperUser {
+  userIdentifier: string
```
Comment (fazanki) [inline src/org-manager/models/organisation.ts]: does this need to be super user or just user

File: src/org-manager/store/effects/organisation.effects.ts [hunk 5]
```diff
@@ -20,9 +21,29 @@ export class OrganisationEffects {
...
+        
+        map(organisationDetails => new organisationActions.LoadOrganisationSuccess(this.mapOrganisations(organisationDetails))),
         catchError(error => of(new organisationActions.LoadOrganisationFail(error)))
       );
     })
-  );
+  )
+
+  mapOrganisations(obj: Organisation[]) : OrganisationVM[] {
+    let organisationModel: OrganisationVM[] = []
```
Comment (fazanki) [inline src/org-manager/store/effects/organisation.effects.ts]: this looks horrid

**Pull request merged**